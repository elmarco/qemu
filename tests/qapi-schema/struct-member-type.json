{ 'struct': 'foo', 'data': { 'a': { 'type': 'str' } } }
