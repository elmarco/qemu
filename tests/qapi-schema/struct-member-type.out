enum QType ['none', 'qnull', 'qnum', 'qstring', 'qdict', 'qlist', 'qbool']
    prefix QTYPE
object foo
    member a: str optional=False
object q_empty
